frontend/rolecall/src/app/piece/piece_editor.component.ts

```diff
@@ -157,12 +160,17 @@ export class PieceEditor implements OnInit {
       }
     }
 
-    const workPieces = pieces.map(piece => ({
-      ...piece,
-      addingPositions: [],
-      originalName: String(piece.name),
-      isOpen: false,
-    }));
+    const workPieces = pieces.map(piece => {
+      if (piece.type === 'SUPER') {
+        this.superDisplay.verifyLoad(piece.uuid, false);
```

Comment:
This should be something like
```
this.balletDisplayService.closeSuperBallet(piece.uuid);
```

Reply:
With the new name this now looks like

this.superBalletDisplay.verifyIsInDisplayList(piece.uuid, false);

The logic is, if the super ballet is not already in the display list, it will be added in the closed state. If is is already in the display list, its open state will not be altered.